src/script.sh

```
@@ -208,55 +225,54 @@ main() {
     fi
 
     ## Setup 
+    ### puts inputs in /home/dnanexus/in/
     dx-download-all-inputs
 
     # Calculate 90% of memory size for java
     MEM=$(head -n1 /proc/meminfo | awk '{print int($2*0.9)}')
     MEM_IN_MB="$(("${MEM}"/1024))m"
 
-    tar zxvf "$fasta_index_path"
+    tar zxvf "$fasta_index_path" -C in/ 
```

Shellcheck (0.10.0)

[warning] fasta_index_path is referenced but not assigned.

(SC2154)